hack/conformance.sh

```diff
@@ -29,6 +29,10 @@ if [[ ! -z "$SKIP_OUTSIDE_CONN_TESTS" ]]; then
     sonobuoy_args="${sonobuoy_args} --plugin-env kubevirt-conformance.E2E_SKIP=\[outside_connectivity\]"
 fi
 
+if [[ ! -z "$RUN_ON_ARM64_INFRA" ]]; then
+    sonobuoy_args="${sonobuoy_args} --plugin-env kubevirt-conformance.E2E_SKIP=.*\[outside_connectivity\].*\[IPv6\].*"
```

Comment:
Looks fine, but note that if the order changes (between the tags) it will not work anymore. I guess a logical "and" would fit better.
But I will not insist on this, it is just fragile.

Reply:
To express the `and` logic would make the regex complex, but I can give it a try.

Reply:
Updated